migrate/sql/20240815115145_add-onchainId-wallet.sql

```diff
@@ -0,0 +1,9 @@
+-- +migrate Up
+ALTER TABLE wallet_ledger ADD onchain_id varchar(128) NULL UNIQUE;
+ALTER TABLE wallet_ledger ADD wallet_signature varchar(128) NULL UNIQUE;
```

Comment:
the `wallet_signature` column isn't necessary. Game server will require wallet signature every single time when users log in with web3 wallet